packages/build/webpack/constants.js

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+const lkModule = process.env.LK_MODULE;
+const lkModuleContainer = process.env.LK_MODULE_CONTAINER;
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+// This path assumes the enlistment in labkey-ui-components is a sibling of the root of the LabKey enlistment.
+const labkeyUIComponentsRelPath = (lkModuleContainer ? "../../../../../" : "../../../../") + "labkey-ui-components/packages/components";
+const labkeyUIComponentsPath = !process.env.LINK ? path.resolve("./node_modules/@labkey/components") : path.resolve(labkeyUIComponentsRelPath);
+console.log("Using @labkey/components path: " + labkeyUIComponentsPath);
+
+const freezerManagerRelPath = (lkModuleContainer ? "../../../../../../" : "../../../../../") + "inventory/packages/freezermanager";
+const freezerManagerPath = !process.env.LINK ? path.resolve("./node_modules/@labkey/freezermanager") : path.resolve(__dirname, freezerManagerRelPath);
```

Comment:
Perhaps switch this so the check doesn't use the negation for better readability?

Reply:
done